### DISCUSSION TITLE
Ideas

### DISCUSSION THREAD

**Discussion by Abhay909** [CubeLabsNZ/CubeTime#68] [Ideas]
1. More fonts for timer and scramble (CS timer font). 

2. Custom backgrounds and color backgrounds. 

3. Change timer size. 

4. Add non WCA events. 

5. Import and export data. Alg trainer. 

6 A share button for sharing with friends. 

If you add all of these it would be the ultimate cubing app.

**Comment by xbjfk** [comment]
Thanks for the suggestions!

1. I suppose we can do this, but this will increase the app size, so we could make these downloadable.
2. This will be quite hard using the iOS apis, maybe in the far future
3. The timer size is already the max that fits on screen with a comfortable margin, but i guess we could add a scale function
4. This depends on the scrambler. We may work on an in-house scrambler and can add these events later, but for now since we use the official WCA tnoodle library this will not be easily possible
5. Import export is in the works and so is alg trainer. no ETA
6. We already have share for results and averages

**Reply by Abhay909** [reply]
Also maybe you could add the app to the App Store on a macbook. Or you could make it a website. Just some ideas

**Reply by pdtxie** [reply]
Once we release the iPadOS version, we will release it for MacOS too (atleast for Apple Silicon macs).

**Reply by pdtxie** [reply]
> Also maybe you could add the app to the App Store on a macbook. Or you could make it a website. Just some ideas

@Abhay909 A MacOS version should be available on Apple Silicon macs :)

**Reply by xbjfk** [reply]
later we might add it for x86 macs with catalyst if i can get tnoodle to work with it

**Comment by EpicGreenMARS** [comment]
Would it be possible to add an import and export feature from cstimer so that it is easier to move your solves across when changing timers?
Also, it would be nice if you could have scrambles that would leave some cfop steps solved so that you could practice a single part, eg OLL if the f2l was already completed 
Another thing is that I think it would be good to have either a web page accessible for mac, or a mac app.
Lastly it would be amazing if you could add stackmat compatibility where possible, or even be  able to use Bluetooth cubes or timers to record times. 
I love your app and I think that others should know about it. You could have an add on cube shops to invite others to use it as well. 
I hope you think these ideas through and I think that the would be amazing to see in a future update.

**Reply by EpicGreenMARS** [reply]
I don't have a silicon mac, so would it be possible to have it in intels?

**Reply by pdtxie** [reply]
These are all wonderful suggestions, thanks! Import/export is currently our highest priority feature, and will hopefully be available soon. Currently the app is only supported on Apple Silicon macs.

**Reply by xbjfk** [reply]
The app is only on arm is due to the way tnoodle is currently compiled, however it may be possible to have x64 catalyst support in the future.
